openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml

```
@@ -19,3 +19,14 @@ allOf:
         required:
           - merchantAccountId
           - apiKey
+      settings:
+        type: object
+        description: MuchBetter settings object
+        properties:
+          brandName:
+            type: string
+            description: MuchBetter brand name use for reporting and logo purpose.
+          extraStep:
```

Boolean properties should start with `is` or `has`.

```suggestion
          hasPhoneNumberRequest:
```